### DISCUSSION TITLE
Why `github-actions` table differs from others managers? `pinDigests` missing sha compare link with github-actions (change)

### DISCUSSION THREAD

**Discussion by bukowa** [renovatebot/renovate#29751] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Any idea why I cannot see compare links and release notes like in Vue here? https://app.renovatebot.com/package-diff?name=vue-tsc&from=1.8.27&to=2.0.21
<details>
  <summary>Click here for Config</summary>
````json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "labels": [
    "dependencies"
  ],
  "osvVulnerabilityAlerts": true,
  "extends": [
    "config:base"
  ],
  "prHourlyLimit": 0,
  "assignees": [
    "bukowa"
  ],
  "gitIgnoredAuthors": [],
  "commitMessageAction": "{{{updateType}}} update",
  "semanticCommits": "enabled",
  "semanticCommitScope": "",
  "semanticCommitType": "deps",
  "commitBodyTable": true,
  "ignoreDeps": [],
  "packageRules": [
    {
      "matchManagers": [
        "github-actions"
      ],
      "groupName": "github actions",
      "labels": [
        "ci",
        "github-actions"
      ],
      "pinDigests": true,
      "semanticCommitType": "ci",
      "semanticCommitScope": "github-actions"
    },
    {
      "matchDatasources": [
        "docker"
      ],
      "labels": [
        "deps",
        "docker"
      ],
      "pinDigests": true,
      "semanticCommitType": "deps",
      "semanticCommitScope": "docker"
    },
    {
      "matchDatasources": [
        "crate"
      ],
      "labels": [
        "deps",
        "rust"
      ],
      "pinDigests": true,
      "semanticCommitType": "deps",
      "semanticCommitScope": "rust"
    },
    {
      "matchDatasources": [
        "npm"
      ],
      "labels": [
        "deps",
        "javascript"
      ],
      "pinDigests": true,
      "semanticCommitType": "deps",
      "semanticCommitScope": "javascript"
    }
  ]
}
````
</details>
<br><br>

![whyy](https://github.com/renovatebot/renovate/assets/38087302/f43309d9-f3a7-40dd-9524-8f370f2eeab2)
![likehere jpog](https://github.com/renovatebot/renovate/assets/38087302/c0bfcc1a-06b5-4f28-8704-0abbe70640ce)

**Comment by rarkins** [comment, marked as answer]
It only works with npm

**Reply by bukowa** [reply]
Any chance to get link to compare on github?
Are there any tracking issues for other managers?
Any plans etc?

**Reply by rarkins** [reply]
No plans currently. I think for GitHub actions you might even achieve it yourself using existing templating of PR columns

**Reply by bukowa** [reply]
> No plans currently. I think for GitHub actions you might even achieve it yourself using existing templating of PR columns

Is there any way to make renovate print the actual config that it  uses?

**Reply by rarkins** [reply]
No

**Reply by BinToss** [reply]
> Is there any way to make renovate print the actual config that it uses?

Yes, but I can't find a way to make it use the config file in an unstaged working tree.

I used...

```sh
npm install -g renovate
npx renovate --print-config --dry-run --token github_pat_**** ${RepoOwner}/${RepoName}
```

...where ${RepoOwner} is bintoss and ${RepoName} is renovate-config to get something like https://pastie.io/ehfhba.json5. The original output is a JSON object without the outermost curly braces and the console output is not commented out.
